src/CheckOrFixXliff/Program.cs

```diff
@@ -130,7 +131,7 @@ private static void FixBrokenTransUnit(XElement tu, XmlNamespaceManager namespac
 			var target = tu.XPathSelectElement("x:target", namespaceManager);
 			if (source.HasElements || target.HasElements)
 			{
-				Console.WriteLine("Cannot fix {0} because the translated material contains XML elements", tuid);
+				Console.WriteLine(@"Cannot fix {0} because the translated material contains XML elements", tuid);
```

Comment:
I don't think this `@` is changing anything (here and throughout).

Reply:
It marks it as verbatim string and so R# knows that it's not a localizable string.

Reply:
Thanks. That's what I thought, but the option to ignore verbatim strings had been turned off in FieldWorks, so I had trained myself to stop doing that.